apps/src/p5lab/AnimationPicker/AnimationPicker.jsx

```diff
@@ -183,39 +199,60 @@ class AnimationPicker extends React.Component {
       console.error('No file found in upload data.');
       return;
     }
-    this.setState({
-      pendingUploadData: data,
-    });
+    if (data.files[0].size >= MAX_UPLOAD_SIZE) {
```

Comment:
I moved this checks for size and file type here from onUploadStart so that if they don't pass, we don't send to Azure image moderator endpoint.